apps/remix-ide/src/app/tabs/locales/zh/udapp.json

```diff
@@ -27,7 +27,7 @@
   "udapp.contractOptionsTitle3": "选择并编译 *.sol 文件以部署或访问合约。",
   "udapp.contractOptionsTitle4": "当有编译的 .sol 文件时，选择 {br} 合约进行部署或与 AtAddress 一起使用。",
   "udapp.checkSumWarning": "您似乎没有使用 checksumed address 。{br} checksumed address 是包含大写字母的地址，如 {a} 中所指定。{br} checksumed address 旨在帮助防止用户将交易发送到错误地址。",
-  "udapp.isOverSizePrompt": "合约创建初始化返回长度超过24576字节的数据。部署可能会失败。 {br}更多信息：{a}",
+  "udapp.isOverSizePromptEip170": "合约创建初始化返回长度超过24576字节的数据。部署可能会失败。 {br}更多信息：{a}",
```

Comment:
You forgot to add translation for `udapp.isOverSizePromptEip3860`

Reply:
I've added the "translation" label, for @ryestew.